src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -3502,8 +3502,6 @@ frame SharedRuntime::look_for_reserved_stack_annotated_method(JavaThread* curren
       if (cb != nullptr && cb->is_nmethod()) {
         nm = cb->as_nmethod();
         method = nm->method();
-        // scope_desc_near() must be used, instead of scope_desc_at() because on
-        // SPARC, the pcDesc can be on the delay slot after the call instruction.
         for (ScopeDesc *sd = nm->scope_desc_near(fr.pc()); sd != nullptr; sd = sd->sender()) {
```

Comment:
It's tempting to try to change this to scope_desc_at(), but also slightly risky if SPARC isn't the only reason it was needed.  Should we investigate this in a separate RFE?